src/scss/components/ads/_ad-control-bar.scss

```
@@ -0,0 +1,66 @@
+@import '../../variables';
+@import '../../mixins';
+
+// Keep this mixin here for now since it is specific to the ad control bar
+@mixin hidden-animated-collapse-and-fade-out($duration: $animation-duration, $original-height: 2em) {
+  transition: opacity $duration ease-out, height $duration ease-out;
+  opacity: 1;
+  height: $original-height;
+
+  &.#{$prefix}-hidden {
+    opacity: 0;
+    height: 0;
+    pointer-events: none;
+  }
+}
+
+.#{$prefix}-ui-controlbar {
```

This should be `ad-controlbar` no? The styles in here are specific for the ad control bar and shouldn't affect the regular `ui-controlbar` as far as I know.


```suggestion
.#{$prefix}-ad-controlbar {
```

---

Yes, let's do that